swaggerpy/async_http_client.py

```diff
@@ -46,36 +46,21 @@ def start_request(self, request_params):
         """
         # request_params has mandatory: method, url, params, headers
         request_params = {
-            'method': str(request_params['method']),
+            'method': str(request_params.get('method', 'GET')),
```

Comment:
We can't assume this is always called from operation.  

The reason for this change is that we're calling it from the Loader, which doesn't necessarily have to pass a method. Even if it does now, it's not required by `HttpClient` so I don't think it should be required here.